### PR TITLE
Added admin endpoints

### PR DESCRIPTION
**Pull request by icellan**

### CODE REVIEW

File: src/transports/http.ts [hunk 1]
```diff
@@ -46,6 +60,191 @@ class TransportHTTP implements TransportService {
...
+        metadata,
+      })
+    });
+  }
+
+  async AdminCreatePaymail(xPubID: string, address: string, public_name: string, avatar: string): Promise<PaymailAddress> {
+    return await this.doHTTPAdminRequest(`${this.serverUrl}/admin/paymail/create`, {
+      method: 'POST',
+      body: JSON.stringify({
+        xPubID,
```
Comment (mrz1836) [inline src/transports/http.ts:168]: Should be `xpub_id`?

**Review by mrz1836 — CHANGES_REQUESTED** [review]
One minor "possible" issue

File: src/transports/http.ts (hunk 1 above)
Comment (icellan) [reply src/transports/http.ts:168]: Yes! Good catch!

**Review by mrz1836 — APPROVED** [review]
LGTM

**Pull request merged**